### PR TITLE
Fix first published date using the wrong date

### PR DESCRIPTION
**Pull request by Tehsurfer**
# Description

Wrike ticket for this sprint 14 ticket can be found here:
https://www.wrike.com/open.htm?id=579586850

Details of changes:

#### Revisions added to the version history pop up. 
View at: https://sparc-pub-date-fix.herokuapp.com/datasets/29?type=dataset
image

#### Original publish date corrected
View at: https://sparc-pub-date-fix.herokuapp.com/datasets/32?type=dataset
image

## Type of change

Delete those that don't apply.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Tested locally and can be viewed at:
https://sparc-pub-date-fix.herokuapp.com/

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: components/VersionHistory/VersionHistory.vue [hunk 1]
```diff
@@ -4,20 +4,23 @@
     :show-close="false"
     class="version-history-dialog"
     width="772px"
-    height="600px"
+    height="360px"
```
Comment (hsorby) [inline components/VersionHistory/VersionHistory.vue:7]: Should these distances be units of `em` or `rem`?

File: components/VersionHistory/VersionHistory.vue [hunk 2]
```diff
@@ -109,7 +114,8 @@ export default {
 <style lang="scss" scoped>
 .version-history-dialog {
   .version-history-container {
-    height: 290px;
+    height: 320px;
```
Comment (hsorby) [inline components/VersionHistory/VersionHistory.vue:117]: `rem` or `em` units?

File: components/VersionHistory/VersionHistory.vue (hunk 1 above)
Comment (Tehsurfer) [reply components/VersionHistory/VersionHistory.vue:7]: Thanks Hugh, happy to switch all of these to `em`

File: components/VersionHistory/VersionHistory.vue (hunk 2 above)
Comment (Tehsurfer) [reply components/VersionHistory/VersionHistory.vue]: Just updated these all to `em` check out my latest commit

**Review by hsorby — APPROVED** [review]

**Pull request merged**